datalad/support/annexrepo.py

```diff
@@ -114,6 +115,14 @@ def __init__(self, path, url=None, runner=None,
           short description that humans can use to identify the
           repository/location, e.g. "Precious data on my laptop"
         """
+
+        if git_opts or annex_opts or annex_init_opts:
+            lgr.warning("TODO: options passed to git, git-annex and/or "
+                        "git-annex-init are currently ignored.\n"
+                        "options received:\n"
+                        "git: %s\ngit-annex: %s\ngit-annex-init: %s" %
+                        (git_opts, annex_opts, annex_init_opts))
```

Comment:
it is the `create` Pr which adds this one here, so the only logical action is to add that passing to super right away to not forget later ;)